health.go

```diff
@@ -890,20 +890,21 @@ type ServerInfo struct {
 
 // MinioInfo contains MinIO server and object storage information.
 type MinioInfo struct {
-	Mode         string       `json:"mode,omitempty"`
-	Domain       []string     `json:"domain,omitempty"`
-	Region       string       `json:"region,omitempty"`
-	SQSARN       []string     `json:"sqsARN,omitempty"`
-	DeploymentID string       `json:"deploymentID,omitempty"`
-	Buckets      Buckets      `json:"buckets,omitempty"`
-	Objects      Objects      `json:"objects,omitempty"`
-	Usage        Usage        `json:"usage,omitempty"`
-	Services     Services     `json:"services,omitempty"`
-	Backend      interface{}  `json:"backend,omitempty"`
-	Servers      []ServerInfo `json:"servers,omitempty"`
-	TLS          *TLSInfo     `json:"tls"`
-	IsKubernetes *bool        `json:"is_kubernetes"`
-	IsDocker     *bool        `json:"is_docker"`
+	Mode         string          `json:"mode,omitempty"`
+	Domain       []string        `json:"domain,omitempty"`
+	Region       string          `json:"region,omitempty"`
+	SQSARN       []string        `json:"sqsARN,omitempty"`
+	DeploymentID string          `json:"deploymentID,omitempty"`
+	Buckets      Buckets         `json:"buckets,omitempty"`
+	Objects      Objects         `json:"objects,omitempty"`
+	Usage        Usage           `json:"usage,omitempty"`
+	Services     Services        `json:"services,omitempty"`
+	Backend      interface{}     `json:"backend,omitempty"`
+	Servers      []ServerInfo    `json:"servers,omitempty"`
+	TLS          *TLSInfo        `json:"tls"`
+	IsKubernetes *bool           `json:"is_kubernetes"`
+	IsDocker     *bool           `json:"is_docker"`
+	Metrics      RealtimeMetrics `json:"metrics"`
```

Comment:
```suggestion
	Metrics      *RealtimeMetrics `json:"metrics"`
```

This will make it easier to parse older versions, since it will be nil.

(it will require reformatting, so you can't apply it here)

Reply:
thanks, done